src/full/Agda/Interaction/Highlighting/LaTeX.hs

```diff
@@ -693,14 +693,12 @@ toLaTeX cc path source hi
 
   . map (second (
       -- Split tokens at newlines
-      concatMap stringLiteral
+      concatMap (stringLiteral
```

Comment:
The structure of function compositions
```haskell
  f
  . g
  . h
```
is destroyed by this fusion.  You now how long ranging parenthesized expressions which are hard to read; the formatting also does not make sense anymore.